site/docs/reference/Connectors/capture-connectors/mixpanel.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebar_position: 1
+---
+# MixPanel
+
+This connector captures data from MixPanel into Flow collections.
+
+It is available for use in the Flow web application. For local development or open-source workflows, [`ghcr.io/estuary/source-mixpanel:dev`](https://ghcr.io/estuary/source-mixpanel:dev) provides the latest version of the connector as a Docker image. You can also follow the link in your browser to see past image versions.
+
+This connector is based on an open-source connector from a third party, with modifications for performance in the Flow system.
+You can find their documentation [here](https://docs.airbyte.com/integrations/sources/mixpanel/),
+but keep in mind that the two versions may be significantly different.
+
+## Supported data resources
+
+The following data resources are supported through the MixPanel APIs:
+
+* [Export](https://developer.mixpanel.com/reference/raw-event-export)
+* [Engage](https://developer.mixpanel.com/reference/engage-query)
+* [Funnels](https://developer.mixpanel.com/reference/funnels-query)
+* [Revenue](https://developer.mixpanel.com/reference/engage-query)
+* [Annotations](https://developer.mixpanel.com/reference/overview-1)
+* [Cohorts](https://developer.mixpanel.com/reference/cohorts-list)
+* [Cohort Members](https://developer.mixpanel.com/reference/engage-query)
+
+By default, each resource is mapped to a Flow collection through a separate binding.
+
+## Prerequisites
+
+* To set up the Mixpanel source connector, you'll need a Mixpanel [Service Account](https://developer.mixpanel.com/reference/service-accounts) and it's [Project ID](https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings#project-id), the [Project Timezone](https://help.mixpanel.com/hc/en-us/articles/115004547203-Manage-Timezones-for-Projects-in-Mixpanel), and the Project region (`US` or `EU`).
+
+## Configuration
+
+You configure connectors either in the Flow web app, or by directly editing the catalog specification file.
+See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about using connectors. The values and specification sample below provide configuration details specific to the MixPanel source connector.
+
+### Properties
+
+#### Endpoint
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| `/project_id` | Project ID | Your project ID number. See the docs for more information on how to obtain this: https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings#project-id | integer | Required |
```

Comment:
Same comment about URLs here. I know this comes from the connector spec but it looks good to reformat the link as text for the docs!